src/angular-business/table/table/table-data-source.ts

```diff
@@ -17,6 +17,33 @@ import { map } from 'rxjs/operators';
  */
 const MAX_SAFE_INTEGER = 9007199254740991;
 
+export const matchesStringCaseInsensitive = (data: string, search: string): boolean => {
+  return data.toLowerCase().indexOf(search.trim().toLowerCase()) !== -1;
```

Comment:
Always use toUpperCase for comparison
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1308-normalize-strings-to-uppercase?view=vs-2015&redirectedfrom=MSDN
https://en.wikipedia.org/wiki/Capital_%E1%BA%9E